pkgs/os-specific/linux/checksec/default.nix

```
@@ -33,18 +40,25 @@ stdenv.mkDerivation rec {
   installPhase =
     let
       path = lib.makeBinPath [
-        findutils
+        binutils
+        coreutils
         file
-        binutils-unwrapped
-        sysctl
+        findutils
+        gawk
+        gnugrep
+        gnused
         openssl
+        procps
+        sysctl
+        which
       ];
     in
     ''
       mkdir -p $out/bin
       install checksec $out/bin
-      substituteInPlace $out/bin/checksec --replace /lib/libc.so.6 ${glibc.out}/lib/libc.so.6
-      substituteInPlace $out/bin/checksec --replace "/usr/bin/id -" "${coreutils}/bin/id -"
+      substituteInPlace $out/bin/checksec --replace "/bin/sed" "${gnused}/bin/sed"
+      substituteInPlace $out/bin/checksec --replace "/usr/bin/id" "${coreutils}/bin/id"
+      substituteInPlace $out/bin/checksec --replace "/lib/libc.so.6" "${glibc}/lib/libc.so.6"
```

```suggestion
      substituteInPlace $out/bin/checksec \
          --replace "/bin/sed" "${gnused}/bin/sed" \
          --replace "/usr/bin/id" "${coreutils}/bin/id" \
          --replace "/lib/libc.so.6" "${glibc}/lib/libc.so.6"
```

substituteInPlace can take multiple `--replace` flags.